Sources/Feather/Feather+DataBases.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  Feather+DataBases.swift
+//  
+//
+//  Created by Tibor Bodecs on 2020. 12. 13..
+//
+
+import FeatherCore
+import LiquidLocalDriver
+import FluentMySQLDriver
+import FluentPostgresDriver
+import FluentSQLiteDriver
+
+extension Feather {
+    
+    ///
+    /// This function will autoconfigure  the instance using system environment  variables
+    ///
+    ///  Using this fonction require the system `ENVIRONEMENT variables` to be set OR an env.`<environement>` file.
+    /// # Reference:
+    /// For more details on how to use env.`<environement>` file, refer to the [Vapor 4 documentation](https://docs.vapor.codes/4.0/environment)
+    ///
+    /// # Required:
+    /// ~~~
+    /// BASE_URL="http://127.0.0.1:8080"
+    /// BASE_PATH="/Repo/feather"
+    /// ~~~
+    /// # Optional:
+    /// ~~~
+    /// MAX_BODYSIZE="10mb" (default) - Required format: XXmb
+    /// PROVIDE_MIDDLEWARE="true" (default) - Required format: true/false
+    ///
+    /// DBTYPE="mysql" # Available:  sqlite (default) / mysql / postgres
+    /// SQL_HOST="127.0.0.1"
+    /// SQL_USER="feather"
+    /// SQL_PASSWORD="feather"
+    /// SQL_DATABASE="feather"
+    ///
+    /// # Optional: For DBTYPE = "mysql" | "postgres"
+    /// SQL_PORT=3306 #  mysql: 3306 (default) - postgres: 5432(default)
+    /// ~~~
+    ///
+
+    /// - Throws: `Error` due to `modules` registration
+    ///
+    /// - parameters:
+    ///     - modules: An Array containing intances of type [ViperBuilder](https://github.com/BinaryBirds/viper-kit)
+    ///
+    public func configureWithEnv(modules userModules: [ViperBuilder] = []) throws {
+        
+        let dbconfig: DatabaseConfigurationFactory
+        let dbID: DatabaseID
+        switch Environment.get("DBTYPE") {
+        case "mysql":
+                dbconfig = .mysql(hostname: Environment.fetch("SQL_HOST"),
```

Comment:
Can we use:
DB_HOST
DB_PORT
DB_USER
DB_PASS
DB_NAME
instead of the SQL prefix please?